imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -80,7 +80,100 @@ func lookForSshdAndGuestAgentProcess() error {
 		return nil
 	}
 
-	return fmt.Errorf("not found")
+	return fmt.Errorf("Guest Agent and/or sshd not found.")
```

Comment:
Whoops it looks like it merged these already. But that's a good idea - I'll open a follow up PR to be clearer about which one is missing.

Windows is a weird case though, since it won't be running sshd